scripts/compile-swarm-extension.sh

```diff
@@ -0,0 +1,11 @@
+SCRIPTS_DIR=$( dirname "$0" )
+
+cd "$SCRIPTS_DIR/.."
+
+git clone https://github.com/ethersphere/swarm-extension.git
```

Comment:
Initially it was an npm dependency. But later it stopped working because it couldn't compile. Probably there was some collision with dependencies. Because of that I removed it from npm dependencies.

It could be a git submodule, but it will add additional complexity. And this script is only used for tests, the swarm extension repo is not needed for the project to compile. So semantically it's not a submodule. I'm not sure, we can do either way.